articles/logic-apps/logic-apps-handle-large-messages.md

```diff
@@ -210,7 +210,13 @@ includes this information about the content that your logic app wants to upload
      |||||
 
 4. After each PATCH request, the endpoint confirms the receipt 
-for each chunk by responding with the "200" status code.
+for each chunk by responding with the "200" status code and the following response headers:
+
+| Endpoint response header field | Type | Required | Description |
```

Comment:
@rohithahw, thanks for making this contribution! Please align the table header row with the leftmost edge of the other rows? The table's left edge should be aligned with the first letter in the preceding paragraph:

![image](https://user-images.githubusercontent.com/13206671/63022550-04e49700-be58-11e9-848a-3aededbe76c3.png)